### DISCUSSION TITLE
Library is deprecated on PHP 8.1

### DISCUSSION THREAD

**Discussion by mkrizanecc** [dbrekelmans/browser-driver-installer#63] [Q&A] [answered]
@dbrekelmans 

Hello, are you planning to update this library to work with PHP 8.1? 

Specificaly, I'm using it with Panther and detect drivers method doesn't work anymore.

**Deprecated: Return type of HumbugBox385\KevinGH\RequirementChecker\RequirementCollection::getIterator() should either be compatible with IteratorAggregate::getIterator(): Traversable, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in phar:///opt/app/vendor/dbrekelmans/bdi/bdi/.box/src/RequirementCollection.php on line 15

Deprecated: Return type of HumbugBox385\KevinGH\RequirementChecker\RequirementCollection::count() should either be compatible with Countable::count(): int, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in phar:///opt/app/vendor/dbrekelmans/bdi/bdi/.box/src/RequirementCollection.php on line 19

Deprecated: Return type of MyCLabs\Enum\Enum::jsonSerialize() should either be compatible with JsonSerializable::jsonSerialize(): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in phar:///opt/app/vendor/dbrekelmans/bdi/bdi/vendor/myclabs/php-enum/src/Enum.php on line 246**

**Comment by dbrekelmans** [comment, marked as answer]
Hi!

Thanks for noting this.

Unfortunately, I'm not going to have time to do this myself in the coming weeks.

If you want to open a pull request with the required changes, I can review it and create a new release for you!
